bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/ChannelConfig.java

```diff
@@ -47,8 +48,8 @@ public class ChannelConfig {
     public boolean trigger = false;
     public String unit = "";
 
-    public String transformationPattern = "";
-    public String transformationPatternOut = "";
+    public @Nullable List<String> transformationPattern;
```

Comment:
Is there a reason to have this `@Nullable`? Wouldn't `public List<String> transformationPattern = List.of()` also work?

Reply:
I've changed it to non nullable